engineering/conventions.md

```diff
@@ -53,4 +54,11 @@ Refer to the [licensing](licensing.md), [maintainers](maintainers.md) and [docum
 
 ## Development Environments
 
-Each sourcerer has his own preferences on editors and IDEs. But if you do not know what to use, [Visual Studio Code](https://code.visualstudio.com/) is a popular choice for Go and Python, and [IntelliJ IDEA](https://www.jetbrains.com/idea/) is the most used for Java and Scala.
\ No newline at end of file
+Each sourcerer has his own preferences on editors and IDEs. But if you do not know what to use, [Visual Studio Code](https://code.visualstudio.com/) is a popular choice for Go and Python, and [IntelliJ IDEA](https://www.jetbrains.com/idea/) is the most used for Java and Scala.
+
+## Configuration
+
+Your application should be [configured](https://12factor.net/config) using environment variables. External services (e.g. databases) URLs, storage paths, [ports to bind](https://12factor.net/port-binding) and logging format all belong to configuration.
```

Comment:
Will be nice include the naming conventions for the environment variables.

Reply:
we (in apps) use a prefix for all our env vars,
example: in `src-d/code-annotation` we choosed `CAT` prefix, so we use `CAT_HOST` instead of `HOST`